dnf-behave-tests/features/metadata.feature

```diff
@@ -24,3 +24,50 @@ Given I copy directory "{context.scenario.repos_location}/dnf-ci-fedora" to "/te
  Then the exit code is 0
   And stdout contains "\s+wget.src\s+1.19.5-5.fc29\s+testrepo"
 
+
+@bz1866504
+Scenario: I cannot create/overwrite a file in /etc from local repository
+# This directory structure is needed at the repo source so that it can be matched on the system running dnf
+# the path where to donwload the file ends up looking something like this:
+# /var/cache/dnf/test-622efad968597580/tmpdir.2fwp3B/../../../../../etc/malicious.file -> /etc/malicious.file
+Given I create directory "a/b/c/d/e/"
```

Comment:
Can you remove the step also in this PR?

Reply:
forgot about this one, removed :slightly_smiling_face: